ArchiSteamFarm/Json/Steam.cs

```diff
@@ -300,26 +300,20 @@ public sealed class ConfirmationDetails : BooleanResponse {
 						return;
 					}
 
-					HtmlDocument htmlDocument = WebBrowser.StringToHtmlDocument(value);
+					IDocument htmlDocument = WebBrowser.StringToHtmlDocument(value).Result;
 
-					if (htmlDocument == null) {
```

Comment:
We should keep the null check, even if we're able to tell this shouldn't generate null result, it's a safeguard worth keeping. Especially since it doesn't hurt us to do that.

Reply:
It's highlighted as "Expression is always false", so null check is completely pointless.

Reply:
Alright then.